src/go/rpk/pkg/cli/cmd/debug/bundle/bundle.go

```diff
@@ -158,9 +161,15 @@ func NewCommand(fs afero.Fs) *cobra.Command {
 	command.Flags().DurationVar(
 		&timeout,
 		"timeout",
-		10*time.Second,
+		15*time.Second,
 		"How long to wait for child commands to execute (e.g. '30s', '1.5m')",
 	)
+	command.Flags().DurationVar(
+		&metricInterval,
```

Comment:
nit: why is the variable metric and the flag metrics? I think plural everywhere makes sense here